examples/user_guide/Lazy_Data.ipynb

```diff
@@ -0,0 +1,107 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Notes (to be removed):\n",
+    "\n",
+    "- What is lazy data?\n",
+    "- Using dask with HoloViews\n",
+    "- Difference between .persist and .compute\n",
+    "  - `persist`: Caching intermediate results but allow further lazy evaluation\n",
+    "  - `compute`: Materialize the data into memory, e.g. ibis->pandas or dask->pandas\n",
+    "- When do you have to consider using persist?\n",
+    "  - Call persist at the end after aggregating or reducing your data in some way\n",
+    "  - Why?\n",
+    "    - Avoids plotting code from evaluating the graph multiple times\n",
+    "- When to consider using compute?\n",
+    "  - Certain interfaces don’t support some methods, e.g. `.sort`, `.iloc`, you might want to perform some operation and then materialize the data into a format that does support these operations\n",
+    "  \n",
+    "Question:  Need examples?"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Lazy Data\n",
+    "\n",
+    "Lazy data, or lazy loading of data, is a technique used in which data is not actually loaded until it is needed.  If used effectively, this allows for a more efficient use of system resources and performance.  Additionally, a user can build up a list of tasks, or build a task graph, that uses the data, but doesn't actually run anything until told to do so.  The *expression* is stored instead of the result.\n",
+    "\n",
+    "Libraries such as Dask use lazy evaluation to their benefit.  Dask will build a task graph of lazy evaluations that it will then run in parallel if it can, depending on compute resources.  This not only speeds things up in time, but it allows computations on larger datasets to be possible when you don't have the memory on your local machine to store the entire dataset during computation.\n",
+    "\n",
+    "Using a library called Ibis, you can manipulate your data using various Ibis expressions, which are lazy, so they are not carried out right away. These expressions will actually be performed using the resources of a remote database instead of your local computer.  \n",
+    "\n",
+    "Lazy expresssions can be quite powerful in that they are essentially making plans to compute something, but packages like Dask or Ibis decide how to compute it.  "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Using Dask with HoloViews"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "As explained here: https://holoviz.org/tutorial/Large_Data.html, HoloViews can accept Dask dataframes just as well as Pandas dataframes.  It has the infostructure to accept the lazy data objects, and will evaluate the objects as needed in order to display the relevant information.  \n",
```

Comment:
I think `infostructure` counts as jargon here, and should be replaced with a more specific description of what you mean. `computational infrastructure`?